PROJECT_GUIDELINES.md

```diff
@@ -0,0 +1,228 @@
+# Project Guidelines
+
+Hey there and thanks for stopping by! We welcome all feedback and contributions to [Calcite App Components](https://github.com/Esri/calcite-app-components/blob/master/README.md). If you are interested in getting involved, we ask that you give our guidelines a quick read through. Below is a list of things we've found work well for us as we continue developing our components.
+
+## TOC
+
+- [Code of conduct](https://github.com/Esri/contributing/blob/master/CODE_OF_CONDUCT.md)
+- [Contribution guidelines](https://github.com/esri/contributing)
```

Comment:
Not sure if this should this be moved under Conventions and link to the contributing section below **or** have the section removed.

Reply:
Maybe move it under conventions.